docs/general/contributing.md

```diff
@@ -92,67 +83,7 @@ compilation errors and that your page is properly rendered in production, we enc
 and render the Wiki on your local machine before submitting a pull request for review. Rendering the
 Wiki locally also adds the benefit of checking for broken links.
 
-### Render the Wiki Locally
-
-The Polkadot Wiki is built from the source files in
-[this GitHub repository](https://github.com/w3f/polkadot-wiki). The Wiki uses Algolia search, which
-can be accessed locally by providing the correct App ID and API key. The `app_id` and `api_key`
-environment variables are needed to build the Wiki successfully. If you are an external contributor,
-set the variables with some values like shown below, which lets the Wiki repo build successfully
-(but disables the search bar).
-
-```bash
-export app_id="xxxxxx" api_key="xxxxxxx"
-```
-
-After cloning the source locally, you can start the website with the commands below (ensure you run
-`yarn` at the root of the repository first to install dependencies).
-
-Using yarn, run:
-
-```bash
-yarn install
-```
-
-Then you can build the Wiki:
-
-```bash
-yarn polkadot:build
-```
-
-And finally, you can start the Wiki:
-
-```bash
-yarn polkadot:start
-```
-
-### Rendering On-chain Values
-
-The Polkadot Wiki can render chain state values via RPC as shown
-[here](https://github.com/w3f/polkadot-wiki/blob/master/components/RPC-Connection.jsx) and display
-them directly in the docs without the need to recompile or even reload the web app. The rendering of
-on-chain values is reserved only in the [Chain State page](chain-state-values.md).
-
-See the example for a react component below:
-
-```bash
-<RPC network="polkadot" path="query.staking.validatorCount" defaultValue="400"/>
-```
-
-Where `network` can be set to `polkadot`, `kusama`, `statemint` (Polkadot Asset Hub), `statemine`
-(Kusama Asset Hub), `polkadotpeople`, and `kusamapeople`. The `path` is composed by:
-
-- `query` or `const`
-- pallet and call
-
-The example above shows the path to query the number of validators within the staking pallet. For
-more information, see
-[pallets and extrinsics](../learn/learn-transactions.md#pallets-and-extrinsics). The `defaultValue`
-is the value the react component shows if there is an issue with fetching data on-chain. A
-[`filter` flag](https://github.com/w3f/polkadot-wiki/blob/master/components/utilities/filters.js) is
-also available to make values human readable, covert block number to days, etc.
-
-## Heritage
-
-This document is based on the
-[Level contribution guidelines](https://github.com/Level/community/blob/master/CONTRIBUTING.md).
+!!!info "Run the Wiki Locally"
+      For more information about how to run the Wiki locally on your machine visit [the README file on GitHub](../../README.md#running-locally).
```

Comment:
So here, we should probably put the HTTP URL. I'll put the "old" Wiki for now, and after it merges it will be updated.

```suggestion
      For more information about how to run the Wiki locally on your machine visit [the README file on GitHub](https://github.com/w3f/polkadot-wiki/blob/master/README.md).
```